entrypoint.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
```

Comment:
I think I understand why this comes up (there was a tough-to-diagnose mismatch between the docker image and a local repo). Depreciating our remote build makes sense to me, given the big hassle it can be when we update gems (and as we continue to move toward maintenance, we're going to be doing a lot more). That said, if people are just doing `docker-compose up` with the build local, they're going to run `bundle install` as part of the regular build, which I think eliminates the need for the `bundle check`, right?

I'm interested in your thoughts here (and a gut check on my assumptions!), but my solution to this would be:

* move the `command` out of docker-compose and into `dockerfile` 
* rather than the entrypoint shell script, have the command be `rails s -p 3000 -b 0.0.0.0`, or whatever the docker version of that is (I think `RUN ['rails', 's', '-p', '3000', '-b', '0.0.0.0']` but I forget exactly)
* keep the local build instead of the image, so that getting the code synced just takes a `git pull` instead of 'and also fight with docker'.

Reply:
This doesn't really deprecate a remote build, it just gives someone an ability to build the image if need be via `docker-compose`. If for instance someone types in `docker-compose up` it will still go remote at first, but this gives someone an opportunity to build locally if they need to.

The entrypoint is meant to just do the following:

* When you type `docker-compose up` just really quickly check with `bundler` if the dependencies are golden. If they aren't, pull them, then start rails. 

Because `docker-compose up` will go remote, that's the reason I setup the entrypoint. You could pull an image with outdated code (since it's copied in as a part of the current build process) and if you've made gemfile updates, this just makes sure that the dependencies are fresh no matter what.

Reply:
I thought about this some and I think I'm good with it, but we're starting to get a lot of root directory docker overhead. Can we move this and `web-variables.env` to their own `docker_util` dir? I think at that point I'm good with this and we can smash merge.

Yr thoughts?

Reply:
In: 294cb8d

Reply:
You can't really move docker-compose (since you need it in root for people to run the command). Everything else is fair game.

Reply:
More in: 3a657f4